.vscode/launch.json

```diff
@@ -12,6 +12,22 @@
       "program": "${file}",
       "console": "integratedTerminal",
       "justMyCode": true
+    },
+    {
+      "name": "Python: SSHNPD",
+      "type": "python",
+      "request": "launch",
+      "cwd": "packages/sshnpdpy/",
+      "program": "sshnpd.py",
+      "console": "integratedTerminal",
+      "justMyCode": true,
+      //replace your arguments with the ones you want to use
+      "args": [
+        "--atsign", "@chess69",
+        "--manager", "@chess69lovely",
+        "--device", "dev",
+        "--username", "xavier"
+      ]
```

Comment:
Is there a way to configure this in a gitignored .env file?

Reply:
Pushed changed for this, let me know if it works...